validator/js/webui/@polymer/webui-mainpage/webui-mainpage.html

```diff
@@ -105,10 +105,25 @@
         // The listbox has the errors at the bottom of the screen.
         var listbox = this.$.errorListBox;
 
+        var isValidatorInited = false;
+
+        if (amp.validator.init) {
+          amp.validator.init().then(function() {
+            isValidatorInited = true;
+            refreshEditorAndErrors();
+          });
+        } else {
+          isValidatorInited = true;
+        }
+
```

Comment:
this code block makes isValidatorInited always true in the end. could you add comment on what it's doing here cause I don't see it from just these changes. is refreshEditorAndErrors called elsewhere when isValidatorInited may be false?

I'd also consider renaming it to isValidatorReady as inited isn't really a word.

Reply:
Renamed it to `isValidatorReady`, and added some comments.